src/lib/settings/ApplicationSettings.cpp

```diff
@@ -337,6 +337,17 @@ void ApplicationSettings::setVerboseIndexerLoggingEnabled(bool value)
 	setValue<bool>("application/verbose_indexer_logging_enabled", value);
 }
 
+FilePath ApplicationSettings::getLogDirectoryPath() const
+{
+	return FilePath(getValue<std::wstring>("application/log_directory_path",
```

Comment:
Sorry, we don't have our coding convention posted anywhere (we just have the clang-format file). Can you please change this call like you formatted `addHelpButton(` in `QtProjectWizardContentPreferences.cpp`?

Reply:
I knew this would come up, eeh sorry 😅
It seems no matter what settings I put in, Qt creator will just convert everything the cursor touches to spaces.

I will fix all these, 
Thanks for the review.

Reply:
Done

Reply:
Hm, no this is still not the convention I was talking about. If a parameter list is broken, all parameters go into the next line, with one tab indentation.

Reply:
Apologies, I misunderstood. Will change it.